devel/apache-arrow/Portfile

```diff
@@ -0,0 +1,164 @@
+                    -DARROW_WITH_SNAPPY=ON \
+                    -DARROW_WITH_BROTLI=ON \
+                    -DARROW_INSTALL_NAME_RPATH=OFF
+    
+# create Python subports
+set python_versions {37 38}
+foreach v ${python_versions} {
+    subport py${v}-py${github.project} {
+        PortGroup   python 1.0
+        python.default_version  ${v}
+    }
+}
+
+if {${name} eq ${subport}} {
+    checksums       rmd160  7c42bb243b87a9ddb8458c43f240fd8d117e40cb \
+                    sha256  b7e0020dd31ae126500ae865248c01bcfc6dbeefbbe223ed0b95339c5f0932c7 \
+                    size    5835020
+
+    # Apache Arrow requires a python3 version to build; use this
+    set python_branch   3.7
+    set python_version  [join [split ${python_branch} .] ""]
+    set python_prefix   ${frameworks_dir}/Python.framework/Versions/${python_branch}
+    set python_bin      ${python_prefix}/bin/python${python_branch}
+
+    depends_build-append \
+                    port:py${python_version}-cython \
+                    port:py${python_version}-numpy
+
+    depends_lib-append \
+                    port:python${python_version}
+
+    # Note: Modifying PATH is required to ensure that cmake finds the correct python version
+    configure.env-append \
+                    "PATH=${python_prefix}/bin:$env(PATH)"
+    configure.dir   ${build_dir}
+
+    build.env-append \
+                    "PATH=${python_prefix}/bin:$env(PATH)"
+    build.dir       ${build_dir}
+    build.target
+
+    destroot.env-append \
+                    "PATH=${python_prefix}/bin:$env(PATH)"
+    destroot.dir    ${build_dir}
+    destroot.cmd    ${build.cmd}
+    destroot.destdir    DESTDIR=${destroot}
+}
+
+# Python bindings for supported Python versions
+if {[string match "py*" ${subport}]} {
+    fetch.type      git
+
+    depends_build-append \
+                    port:py${python.version}-cython \
+                    port:py${python.version}-pip \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-setuptools_scm \
+                    port:py${python.version}-wheel
+
+    depends_lib-append \
+                    port:apache-arrow \
+                    port:py${python.version}-numpy
+
+    depends_test-append \
+                    port:py${python.version}-cffi \
+                    port:py${python.version}-hypothesis \
+                    port:py${python.version}-pandas \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-pytest-lazy-fixture \
+                    port:py${python.version}-tz
+
+    post-fetch {
+        system -W ${worksrcpath} "git submodule update --init --recursive"
+    }
+
+    configure.pre_args-delete \
+                    --prefix=${prefix}
+
+    build.env-append \
+                    "PATH=${python.prefix}/bin:$env(PATH)" \
+                    PYARROW_CMAKE_OPTIONS=${configure.pre_args} \
+                    PYARROW_WITH_PARQUET=1    
```

Comment:
There is trailing whitespace on this line.

Reply:
Thanks fixed.